File: hospital/AMarotoChivite-HospitalTest/AMarotoChivite-HospitalTest-Java/Ejercicios_Intermedios_JavaMaven/src/test/java/HospitalCollections/repositories/MapImplementacionRepositoryTest.java
package HospitalCollections.repositories;

import org.junit.jupiter.api.BeforeEach;

public class MapImplementacionRepositoryTest extends RepositoryTest {

    @BeforeEach
    public void setUp() {
        // Inicializa la implementación específica de la interfaz
        this.repoToEspecify = new PatientMapRepository();
    }

    // Prueba específicas si lo necesito...
    // Utilizo this.repoToEspecify el tipo de repo...
}

File: hospital/AMarotoChivite-HospitalTest/AMarotoChivite-HospitalTest-Kotlin/src/test/kotlin/HospitalCollectionsTest/repositories/MapImplementacionRepositoryTest.kt
package HospitalCollectionsTest.repositories

import HospitalCollections.base.IHospitalRepository
import HospitalCollections.repositories.PatientMapRepository

class MapImplementacionRepositoryTest : RepositoryTest() {
    // Inicializa la implementación específica de la interfaz
    override var repoToEspecify: IHospitalRepository = PatientMapRepository()

    // Pruebas específicas si lo necesito...
    // Utilizo this.repoToEspecify el tipo de repo...
}